exercises/word-count/src/example.c

```diff
@@ -0,0 +1,75 @@
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+#include "word_count.h"
+
+// checks to see if we are counting a given word:
+//    if word counted already it returns index.
+//    elseo returns -1
+static int word_match(char *testWord, int current_word_count, WordCount_Word_t words[MAX_WORDS])
+{
+   int index;
+   int matchIndex = -1;
+
+   for (index = 0; index < current_word_count; index++)
+   {
+      if (!strcmp(testWord, words[index].text))
+      {
+         matchIndex = index;
+         break;
+      }
+   }
+   return matchIndex;
+}
+
+int word_count(char *input_text, WordCount_Word_t words[MAX_WORDS])
+{
+   int index;
+   int uniqueWords = 0;
+   char *testWord;
+   char *lowerCaseInputText;  // hold lowercase copy of passed text
+   const char *delimiters = " ,.-\n:!!&@$%^&\"";
+
+   // convert the input text to lower case
+   lowerCaseInputText = malloc(strlen(input_text)+1);
+   for(index=0; input_text[index] != 0; index++)
+   {
```

Comment:
To make things easier, you can just run `indent.sh` in the root directory of the repo to automatically apply the coding style.